internal/pkg/rbldap/generate.go

```diff
@@ -0,0 +1,38 @@
+package rbldap
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/redbrick/rbldap/pkg/rbuser"
+	"github.com/urfave/cli"
+)
+
+// Generate takes cli context and generates user vhost for rbuser
+func Generate(ctx *cli.Context) error {
+	l, err := newRBLdap(ctx)
```

Comment:
This looks very weird. I would change it to `newRbLdap`. Still not ideal but at least consistent.

Reply:
how is `Rb` consistent compared to `RB`

Reply:
If you only capitalise `L` in `Ldap` then only `R` should be capitalised in `Rb`.

Reply:
hmmm well fine then

Reply:
what about `newDCULdap` then?